### PR TITLE
Update DP status tag localisations

### PR DESCRIPTION
**Pull request by ethax-ross**
[Jira-2564](https://dfedigital.atlassian.net/browse/CPDLP-2564)

### Context

We have a new set of copy for some of the record states in the delivery partner status tag. We also want to remove the training status column (to leave only the status column).

### Changes proposed in this pull request

- Update DP status tag localisations

We have a new set of copy for some of the record states in the delivery partner status tag.

- Remove training status column from DP participants

We no longer want to display the training status in the DP participants view.

### Guidance to review

I've added a bunch of participants to [this delivery partner](https://cpd-ecf-review-4270-web.test.teacherservices.cloud/delivery-partners/6cdcdf5c-3ab0-4759-b67c-f216490d9405/participants) for testing.

### CODE REVIEW

File: config/locales/status_tags/delivery_partner_participant_status.yml [hunk 1]
```diff
@@ -27,168 +27,168 @@ en:
       internal_error:
         id: "dfe_checking_eligibility"
         label: "DfE checking eligibility"
-        description: "We’re checking these people’s details with the Teaching Regulation Agency to make sure they’re eligible for funding. We’ll confirm this soon."
```
Comment (cwrw) [inline config/locales/status_tags/delivery_partner_participant_status.yml:30]: could you add `validation_not_started` amendment as well?

File: config/locales/status_tags/delivery_partner_participant_status.yml [hunk 2]
```diff
@@ -27,168 +27,168 @@ en:
...
         id: "not_eligible_for_funded_training"
         label: "Not eligible for funded training"
-        description: "We’ve checked these people’s details and found they’re not eligible for this programme."
+        description: "We’ve checked this participant’s details and found they’re not eligible for this programme."
 
       not_qualified:
         id: "checking_qts"
         label: "Checking QTS"
-        description: "These ECTs do not have qualified teacher status (QTS) yet. They need this to be eligible for funded training. We’ll keep checking their status and notify you if something changes."
+        description: "This ECT does not have qualified teacher status (QTS) yet. They need this to be eligible for funded training. We’ll keep checking their status and notify you if something changes."
```
Comment (cwrw) [inline config/locales/status_tags/delivery_partner_participant_status.yml:72]: not sure this matches the document, worth confirming

File: config/locales/status_tags/delivery_partner_participant_status.yml [hunk 3]
```diff
@@ -27,168 +27,168 @@ en:
...
-        description: "We’ve confirmed these people are eligible for this programme. Your training provider will contact them directly."
+        description: "We’ve confirmed this participant is eligible for this programme. Your training provider will contact them directly."
 
       deferred_training:
-        id: "training_or_eligible_for_training"
-        label: "Training or eligible for training"
-        description: "We’ve confirmed these people are eligible for this programme. Your training provider will contact them directly."
+        id: "participant_deferred"
+        label: "Participant deferred"
+        description: "Your provider has reported that this particpant has deferred. Contact the provider if you think this is wrong."
```
Comment (cwrw) [inline config/locales/status_tags/delivery_partner_participant_status.yml:157]: worth checking with Mark if the content on the design v2 is the latest or the document as they are slightly different

**Review by cwrw — COMMENTED** [review]
looks good, just some content qs otherwise 👍

**Comment by ethax-ross** [comment]
@cwrw I caught up with Mark as there were some changes from the original spreadsheet; he's checked it over now and it should be good.

**Review by cwrw — APPROVED** [review]

**Pull request merged**